app.js

```diff
@@ -1,6 +1,61 @@
 const express = require('express');
+const mongoose = require('mongoose');
+const bodyParser = require('body-parser');
+const keys = require('./config/keys');
+require('./models/Property');
+require('./models/Owner');
+const Property = mongoose.model('property');
```

Comment:
let's not do it this way, let's do it in a more cleaner/readable way, starting with exporting the models/schema from the model files themselvs

Reply:
Done